keyboards/skyloong/gk75/q1/q1.c

```diff
@@ -0,0 +1,267 @@
+//  Copyright 2023 JZ-Skyloong (@JZ-Skyloong)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+_Bool FN_WIN = 0;
+_Bool FN_MAC = 0;
+_Bool L_WIN = 0;
+_Bool L_MAC = 0;
+
+#    ifdef RGB_MATRIX_ENABLE
+const is31fl3743a_led_t PROGMEM g_is31fl3743a_leds[IS31FL3743A_LED_COUNT] = {
+/* Refer to IS31 manual for these locations
+ *   driver
+ *   |  R location
+ *   |  |          G location
+ *   |  |          |          B location
+ *   |  |          |          | */
+    {0, CS1_SW1,   CS2_SW1,   CS3_SW1},
```

Comment:
These should be swapped around (SWx_CSy etc).

Reply:
OK, I changed it.